client/components/TestPlanVersionsPage/index.jsx

```diff
@@ -273,10 +273,48 @@ const TestPlanVersionsPage = () => {
                                                     </PhasePill>
                                                 );
 
+                                                const draftPill = (
+                                                    <PhasePill
+                                                        fullWidth={false}
+                                                    >
+                                                        DRAFT
+                                                    </PhasePill>
+                                                );
+
+                                                if (
+                                                    derivedDeprecatedAtPhase ===
+                                                    'RD'
+                                                ) {
+                                                    return (
+                                                        <>
+                                                            {deprecatedPill}
+                                                            before&nbsp;
+                                                            {draftPill}
+                                                            review
+                                                        </>
+                                                    );
+                                                }
+
+                                                if (
+                                                    derivedDeprecatedAtPhase ===
+                                                    'RECOMMENDED'
+                                                ) {
+                                                    return (
+                                                        <>
+                                                            {deprecatedPill}
+                                                            after being approved
+                                                            as&nbsp;
+                                                            {phasePill}
+                                                        </>
+                                                    );
+                                                }
+
                                                 return (
                                                     <>
-                                                        {deprecatedPill} during{' '}
-                                                        {phasePill} review
+                                                        {deprecatedPill}
```

Comment:
With one that has been deprecated in Candidate, NVDA announces it as `Latest Phase  column 2  Deprecatedduring Candidatereview`. The same doesn't happen with VO for me.